sparse/core.py

```diff
@@ -620,32 +611,85 @@ def __pow__(self, other):
         return self.elemwise(operator.pow, other)
 
     def __and__(self, other):
-        return self.elemwise_binary(operator.and_, other)
+        return self.elemwise(operator.and_, other)
 
     def __xor__(self, other):
-        return self.elemwise_binary(operator.xor, other)
+        return self.elemwise(operator.xor, other)
 
     def __or__(self, other):
-        return self.elemwise_binary(operator.or_, other)
+        return self.elemwise(operator.or_, other)
+
+    def __gt__(self, other):
+        return self.elemwise(operator.gt, other)
+
+    def __ge__(self, other):
+        return self.elemwise(operator.ge, other)
+
+    def __lt__(self, other):
+        return self.elemwise(operator.lt, other)
+
+    def __le__(self, other):
+        return self.elemwise(operator.le, other)
+
+    def __eq__(self, other):
+        return self.elemwise(operator.eq, other)
+
+    def __ne__(self, other):
+        return self.elemwise(operator.ne, other)
 
     def elemwise(self, func, *args, **kwargs):
+        """
+        Apply a function to one or two arguments.
+
+        Parameters
+        ----------
+        func
+            The function to apply to one or two arguments.
+        args : tuple, optional
+            The extra arguments to pass to the function. If args[0] is a COO object
+            or a scipy.sparse.spmatrix, the function will be treated as a binary
+            function. Otherwise, it will be treated as a unary function.
+        kwargs : dict, optional
+            The kwargs to pass to the function.
+        Returns
```

Comment:
I think we need a newline above this docstring title for clean rendering with sphinx